zip-guide.rst

```diff
@@ -0,0 +1,127 @@
+::
+
+  ZIP: Unassigned {numbers are assigned by ZIP editors}
+  Title: {Something Short and To the Point}
+  Owners: First Owner <email>
```

Comment:
Many of the current ZIPs use `Author`, so which is correct?

Reply:
> Many of the current ZIPs use `Author`, so which is correct?

See https://github.com/zcash/zips/blob/master/zip-0000.rst#zip-header-preamble - it was updated to `Owner`

Reply:
``Owners`` is correct. I'll update the existing ZIPs.